python/ThirteenTeV/Hadronizer/Hadronizer_TuneCP5_13TeV_pTmaxMatch_1_pTmaxFudge_oneoversqrt2_LHE_pythia8_cff.py

```diff
@@ -9,15 +9,15 @@
     filterEfficiency = cms.untracked.double(1.0),
     pythiaHepMCVerbosity = cms.untracked.bool(False),
     comEnergy = cms.double(13000.),
-    processParameters = cms.vstring(
```

Comment:
isn't this commit redundant?

Reply:
@perrozzi previously it wasn't working because processParameters were outside PythiaParameters, they are supposed to be inside.